specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,710 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+      "title": "Azure SignalR Service REST API",
+      "version": "v1"
+    },
+    "paths": {
+      "/api/v1/health": {
+        "get": {
+          "description": "Get service health status",
```

Comment:
This is actually a summary

Reply:
Are there any rules for distinguishing `summary` and `description`? Does it matter if it is `description`? Actually the swagger is auto-generated from `Swashbuckle Swagger` and the `description` is extracted from **triple-slash comments**'s `<summary>`  directly.

Reply:
It's not critical, since most of the SDK would be fine if they have either a summary or a description. If you auto-generates this file and there is no configuration, it's not worth the effort.

Reply:
Updated to summary